src/apiCalls.js

```diff
@@ -0,0 +1,62 @@
+let fetchCustomers = () => {
```

Comment:
Had these as variables in whats-cooking project but after looking over the advent work on api calls, I feel like this is the better way to go.

On the advent, we wrote the functions with a let rather than const. Wonder if that's the most correct.